locales/fr.json

```
@@ -0,0 +1,128 @@
+{
+    "locale": {
+        "code": "fr",
+        "displayName": "French",
+        "nativeDisplayName": "Français",
+        "emoji": "🇫🇷"
+    },
+    "common": {
+        "documentation": "Documentation",
+        "loading": "Chargement ...",
+        "download": "Téléchargement",
+        "releases": "Versions",
+        "allVersions": "Toutes les Versions",
+        "sourceCode": "Code Source",
+        "faq": "FAQ",
+        "error": "Erreur",
+        "contributors": "Contributeurs",
+        "contributorsDescription": "vrc-get et ALCOM sont des projets open-source crées grace aux contributions de nombreux développeurs. Voici quelques contributeurs du project :"
+    },
+    "downloads": {
+        "macOS": "DiskImage macOS",
+        "linux": "Linux",
+        "windowsSetup": "Windows Installer",
+        "windowsStandalone": "Windows Standalone",
+        "unsuportedOS": "OS non supporté"
+    },
+    "releases": {
+        "viewOnGitHub": "Voir sur GitHub",
+        "assets": "Assets",
+        "noChangelogMessage": "Pas de changelog"
+    },
+    "home": {
+        "productPage": "Page Produit",
+        "alcomDescription": "Une alternative rapide et open-source au VRChat Creator Companion",
+        "cliDescription": "Une ligne de commande rapide et open-source, client pour le VRChat Package Manager"
+    },
+    "alcom": {
+        "hero": "Une alternative rapide et open-source au VRChat Creator Companion (VCC)<br>ALCOM est une application cross-platform disponible sur Windows, macOS, et Linux.",
+        "promotions": {
+            "__comment for translators": "There is no need to translate the each promotion exactly. You can translate it in a way that makes sense in your language as long as matches with image / video. Especially for internationalized, you can say 'Your Language is supported'. (I actually do for Japanese)",
+            "__comment for translators2": "You don't have to create new image / video for each language, you can use english. (I use english version for Japanese). When you add, please add to `/resources/alcom-assets/<language>/<same-name.png>`.",
+            "fastInstallation": {
+                "video": "/resources/alcom-assets/installing-packages.webm",
+                "title": "Installation fluide et rapide",
+                "content": "<p>Profitez d'une installation significativement plus rapide de vos packages. Installez plusieurs packages simultanément et profitez de mises a jour rapides en quelques clics.</p>"
+            },
+            "accessToChangelog": {
+                "image": "/resources/alcom-assets/changelog-shown.png",
```

```suggestion
                "image": "/resources/alcom-assets/fr/changelog-shown.png",
```

actually there already is images for all languages available for ALCOM